examples/sveltekit-houdini/src/client.ts

```diff
@@ -0,0 +1,13 @@
+import { HoudiniClient } from '$houdini';
+import { PUBLIC_API_URL, PUBLIC_API_KEY } from '$env/static/public'
```

Comment:
`.env.example` has different keys 
 ```suggestion
import { PUBLIC_GRAFBASE_API_URL, PUBLIC_GRAFBASE_API_KEY } from '$env/static/public'
```